fixtures/sample_transistive_static/.gitignore

```diff
@@ -0,0 +1,63 @@
+### macOS ###
```

Comment:
Isn't enough with the one at the root directory?

Reply:
Yes, I believe this was generated. Will remove.